community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/MultipleIndexPopulator.java

```diff
@@ -317,9 +333,14 @@ private IntStream propertyKeyIds( IndexPopulation population )
         return IntStream.of( population.schema().getPropertyIds() );
     }
 
-    private int[] labelIds()
+    private int[] entityTokenIds()
     {
-        return populations.stream().mapToInt( population -> population.schema().getLabelId() ).toArray();
+        if ( populations.stream().anyMatch( indexPopulation -> indexPopulation.schema().getEntityTokenIds().length == 0 ) )
+        {
+            //No token is any token
```

Comment:
this should not be here, empty array and [0] element of array is leaking everywhere

Reply:
Do you have any proposals for better defining any label/reltype than the empty array? I saw it being defined as empty array somewhere in existing code, and decided to follow that pattern since I couldn't figure out anything better.

Reply:
Replaced with a constant for now, feels a bit better at least.